docs/platform/howto/manage-org-vpc-peering-azure.md

```diff
@@ -409,11 +464,12 @@ from your Azure VNet to the Aiven organization VPC:
 1. Check if the status of the peering connection is `ACTIVE`:
 
    ```bash
-   avn vpc peering-connection get -v             \
-     --project-vpc-id $aiven_organization_vpc_id \
-     --peer-cloud-account $user_subscription_id  \
-     --peer-resource-group $user_resource_group  \
-     --peer-vpc $user_vnet_name
+   avn organization vpc get -v                       \
```

Comment:
Here too, the command does not have `-v` flag as there's so far been no need to define a more verbose output format. A complete invocation of `organization vpc get` looks like this:
```
avn organization vpc get --organization-id org4f9ed964bca --organization-vpc-id 6a5542cf-2a10-416e-80c2-dcaeddc99a9a
```